student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Please always double-check your work before submitting it for review. as I noticed that I created the object for the `student.rb` file and as soon as I printed the value of the student name, I got an error for the parameter that I passed for the student, which further passed the parameters to the `super` user which is person in your case. Kindly resolve the error first and submit your project again without any error.

![image](https://github.com/ivangonzalez224/School-library/assets/66476329/acb8250d-9f5f-41ce-8e85-aa9500841fdc)

Reply:
Hi @skyv26 thank you for the review.
Changes done, I updated the arguments in the classes.